src/main/java/DukeExceptions.java

```diff
@@ -0,0 +1,2 @@
+public class DukeExceptions extends Exception{
```

Comment:
Perhaps you could leave a space here before `{` for layout consistency. Similar styles also happen in line 3 of DeadlineException class.